src/types.ts

```diff
@@ -0,0 +1,352 @@
+    blobs_url: string;
+    branches_url: string;
+    clone_url: string;
+    collaborators_url: string;
+    comments_url: string;
+    commits_url: string;
+    compare_url: string;
+    contents_url: string;
+    contributors_url: string;
+    deployments_url: string;
+    downloads_url: string;
+    events_url: string;
+    forks_url: string;
+    git_commits_url: string;
+    git_refs_url: string;
+    git_tags_url: string;
+    git_url: string;
+    hooks_url: string;
+    issue_comment_url: string;
+    issue_events_url: string;
+    issues_url: string;
+    keys_url: string;
+    labels_url: string;
+    languages_url: string;
+    merges_url: string;
+    milestones_url: string;
+    mirror_url: string;
+    notifications_url: string;
+    pulls_url: string;
+    releases_url: string;
+    ssh_url: string;
+    stargazers_url: string;
+    statuses_url: string;
+    subscribers_url: string;
+    subscription_url: string;
+    svn_url: string;
+    tags_url: string;
+    teams_url: string;
+    trees_url: string;
+    homepage: string;
+    language?: any;
+    forks_count: number;
+    stargazers_count: number;
+    watchers_count: number;
+    size: number;
+    default_branch: string;
+    open_issues_count: number;
+    topics: string[];
+    has_issues: boolean;
+    has_wiki: boolean;
+    has_pages: boolean;
+    has_downloads: boolean;
+    pushed_at: Date;
+    created_at: Date;
+    updated_at: Date;
+    permissions: Permissions;
+    allow_rebase_merge: boolean;
+    allow_squash_merge: boolean;
+    allow_merge_commit: boolean;
+    subscribers_count: number;
+    network_count: number;
+}
+
+export interface Issue {
+    id: number;
+    url: string;
+    repository_url: string;
+    labels_url: string;
+    comments_url: string;
+    events_url: string;
+    html_url: string;
+    number: number;
+    state: string;
+    title: string;
+    body: string;
+    user: User;
+    labels: Label[];
+    assignee: Assignee;
+    milestone: Milestone;
+    locked: boolean;
+    comments: number;
+    pull_request: PullRequest;
+    closed_at?: any;
+    created_at: Date;
+    updated_at: Date;
+    repository: Repository;
+}
+
+export interface Comment {
+    id: number;
+    url: string;
+    html_url: string;
+    body: string;
+    user: User;
+    created_at: Date;
+    updated_at: Date;
+}
+
+export interface Sender {
+    login: string;
+    id: number;
+    avatar_url: string;
+    gravatar_id: string;
+    url: string;
+    html_url: string;
+    followers_url: string;
+    following_url: string;
+    gists_url: string;
+    starred_url: string;
+    subscriptions_url: string;
+    organizations_url: string;
+    repos_url: string;
+    events_url: string;
+    received_events_url: string;
+    type: string;
+    site_admin: boolean;
+}
+
+export interface WebhookEvent {
+    action: string;
+    issue: Issue;
+    repository: Repository;
+    label: Label;
+    sender: Sender;
+}
+
+export interface Milestone {
```

Comment:
There are two `export interface Milestone` declarations in this file.  Possibly not the only duplicates?